resources/icons/16px/inherited-group.svg

```diff
@@ -2,13 +2,12 @@
044605e-15,8 C-2.22044605e-15,7.45 0.45,7 1,7 Z M6.38756714,0 C7.01699253,0 7.72634495,0.21 8.22588891,0.6 C8.70545111,0.98 9.07511364,1.51 9.24495859,2.12 C9.2849221,2.25 9.31489474,2.39 9.3348765,2.52 C9.42479441,3 9.47474881,3.74 9.47474881,4.25 L9.47474881,4.32 C9.65458463,4.4 9.8144387,4.59 9.84441134,4.99 C9.87438398,5.31 9.75449343,5.58 9.68455727,5.7 C9.624612,5.98 9.47474881,6.28 9.20499507,6.33 C9.17502243,6.46 9.13505892,6.59 9.08510452,6.72 C9.08510452,6.73 9.07511364,6.76 9.07511364,6.76 C8.8553143,7.34 8.52561529,7.84 8.12598012,8.21 L8.12598012,8.39 C8.16594363,8.84 8.00608957,9.16 9.18501331,9.69 C10.3639371,10.22 12.1323227,10.78 12.4919943,11.64 C12.8616569,12.5 12.7117937,13 12.7117937,13 L9,13 L9,13 C9,12.4610272 8.78911376,11.9549002 8.41710678,11.5828932 L6.41710678,9.58289322 C6.0450998,9.21088624 5.53897282,9 5,9 C4.85114495,9 4.70385647,8.98498907 4.55813455,8.95496721 C4.65673966,8.76543141 4.64044866,8.58584795 4.65914504,8.38 C4.65914504,8.33 4.66913592,8.27 4.66913592,8.21 C4.25950987,7.86 3.91981998,7.35 3.70002064,6.76 L3.70002064,6.75 C3.70002064,6.75 3.69002976,6.74 3.69002976,6.73 C3.65006624,6.61 3.60011185,6.47 3.57013921,6.34 C3.29039459,6.29 3.13054052,5.98 3.07059525,5.7 C3.01064997,5.58 2.88076854,5.31 2.91074118,4.99 C2.9507047,4.58 3.12054964,4.39 3.30038547,4.31 L3.30038547,4.25 C3.30038547,3.74 3.35033987,2.99 3.44025778,2.51 C3.46023954,2.38 3.49021217,2.24 3.53017569,2.11 C3.70002064,1.51 4.06968317,0.98 4.54924537,0.6 C5.04878933,0.21 5.75814175,0 6.38756714,0 Z M11.0133442,2.04 C11.5028973,2.04 12.0623865,2.2 12.4520308,2.5 C12.8316842,2.79 13.1214197,3.2 13.2513011,3.67 C13.2812738,3.77 13.3012555,3.88 13.3212373,3.98 C13.3911734,4.35 13.431137,4.92 13.431137,5.31 L13.431137,5.36 C13.5710093,5.42 13.7008907,5.57 13.7208725,5.87 C13.7408542,6.12 13.6509363,6.32 13.590991,6.41 C13.5410366,6.62 13.431137,6.85 13.2113376,6.89 C13.1913559,6.99 13.1613832,7.09 13.1214197,7.19 C13.1214197,7.2 13.1114288,7.22 13.1114288,7.22 C12.9415839,7.66 12.681821,8.05 12.3621129,8.33 L12.3621129,8.47 C12.3920855,8.82 12.272195,9.06 13.1913559,9.47 C14.1205076,9.88 15.5092398,10.31 15.7889845,10.97 C16.07872,11.63 15.9588294,12.01 15.9588294,12.01 L13.6609272,12.01 L13.6609272,12.02 C13.6109728,11.78 13.5210549,11.52 13.4111552,11.26 C13.0514836,10.4 12.2921767,9.93 10.7236087,9.26 C10.5837364,9.2 10.1341468,9.01 10.124156,9.01 C9.91434749,8.92 9.7644843,8.86 9.624612,8.79 C9.64459375,8.69 9.64459375,8.59 9.65458463,8.48 C9.65458463,8.44 9.66457551,8.4 9.66457551,8.35 C9.59463936,8.29 9.53469408,8.23 9.47474881,8.16 C9.69454815,7.84 9.87438398,7.49 10.0142563,7.11 C10.034238,7.05 10.034238,7.05 10.0442289,7.01 C10.3339644,6.78 10.5237911,6.44 10.6336908,6.05 C10.7935449,5.72 10.8834628,5.32 10.8434993,4.89 C10.8135266,4.49 10.6836452,4.13 10.4738367,3.86 C10.453855,3.33 10.4039006,2.73 10.3239735,2.32 C10.3139827,2.26 10.3039918,2.2 10.2940009,2.13 C10.5237911,2.07 10.7735631,2.04 11.0133442,2.04 Z" id="path-1"></path>
```

Comment:
this works, but it departs from the existing icons which don't use any `defs`... for example https://github.com/palantir/blueprint/blob/develop/resources/icons/16px/inheritance.svg?short_path=85288f3

it's a nitpick, but if it's not too much of a hassle, could you export the shape as a plain path without `<use>`?